components/profile/rating/src/index.scss

```diff
@@ -0,0 +1,30 @@
+@import '~@schibstedspain/theme-basic/lib/index';
+
+$fill-profile-rating-filled: #f7af33 !default;
```

Comment:
We used to set default Sass values [here](https://github.com/SUI-Components/sui-theme/blob/master/src/settings-compat-v7/_components.scss) and then the corporate theme rewrite these values.
Since we are not doing this any more it would be nice to have Sass Vars declarations inside a separate partial file like:
`_settings.scss`

Reply:
I asumed talking with @JoanClaret and @frmedina that there's a new way to declare component variables that is inside the component.

Reply:
This is exactly what I'm saying. Since we have a new way to declare Sass variables inside the component, use at least a separate partial file called `_settings.scss`